status-im/status-legacy
# [WIP] Fix for Issue #8718

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

Attempts to fix #8718 

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
Attempts to solve the issue where some languages do not display correctly and default to English.

### Review notes


### Testing notes


#### Platforms


- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats
- wallet / transactions
- dapps / app browsing
- account recovery
- new account
- user profile updates
- networks
- mailservers
- fleet
- bootnodes

##### Non-functional

- battery performance
- CPU performance / speed of the app
- network consumption

### Steps to test

Step 1, configure your device to a different language
Step 2, launch Status
Step 3, verify Status translates to the correct language



status: wip
